nilearn/datasets/struct.py

```diff
@@ -117,14 +123,15 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
 
 
 def load_mni152_template():
-    """Load skullstripped 2mm version of the MNI152 originally distributed
```

Comment:
here we could have `load_mni152_template(resolution_mm=2)`, as for the difumo atlas for example

Reply:
I am not sure whether I understand your suggestion here. We are simply loading a specific template here, which is now the new one with 1mm resolution. Do you want to add a subroutine in this function that also re-samples the template in case the resolution parameter is not 1mm?

Reply:
I think @jeromedockes is right here. Currently, you can change the resolution through `compute_brain_mask` but you might want to do that when loading the template itself. 
Also in terms of backward compatibility, previous code expecting a 2mm resolution image as the output of these template loading functions might break if we don't resample here. Does this make sense?